Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/qrcode/scan/SubmissionQRCodeScanFragment.kt

```diff
@@ -59,21 +60,29 @@ class SubmissionQRCodeScanFragment : Fragment(R.layout.fragment_submission_qr_co
         binding.submissionQrCodeScanViewfinderView.setCameraPreview(binding.submissionQrCodeScanPreview)
 
         viewModel.scanStatus.observeEvent(viewLifecycleOwner) {
```

Comment:
as this is only observed here and the value does not matter we could also replace scan status with a `SingleLiveEvent<Unit>`

Reply:
this has been implmented. please check it out.